src/test/crimson/seastore/test_object_data_handler.cc

```diff
@@ -671,6 +678,34 @@ TEST_P(object_data_handler_test_t, multiple_overwrite) {
   });
 }
 
+TEST_P(object_data_handler_test_t, random_overwrite) {
+  constexpr size_t TOTAL = 4<<20;
+  constexpr size_t BSIZE = 4<<10;
+  constexpr size_t BLOCKS = TOTAL / BSIZE;
+  run_async([this] {
+    set_overwrite_threshold();
```

Comment:
For the delta-based overwrite test cases, proper asserts are needed to make sure that the write sizes we are testing must be smaller than the threshold, to prevent future arbitrary modifications in making the tests invalid by accident.

Reply:
Done.

Reply:
I improved this test case, including applying the check for threshold, please see the recent change.